test/e2e/lazy_activation_policy_test.go

```diff
@@ -192,6 +193,22 @@ var _ = ginkgo.Describe("Lazy activation policy testing", func() {
 			})
 		})
 
+		// Combined Case 3 (Policy preemption)
+		// refer: https://github.com/karmada-io/karmada/blob/release-1.9/docs/proposals/scheduling/activation-preference/lazy-activation-preference.md#combined-case-3-policy-preemption
+
+		ginkgo.It("Combined Case 3 (Policy preemption)", func() {
+			ginkgo.By("step 1: deployment propagate success when policy created before it", func() {
+				waitDeploymentPresentOnCluster(originalCluster, namespace, deploymentName)
+			})
+
+			ginkgo.By("step 2: create PP2 (match nginx, cluster=member2, lazy, priority=2, preemption=true)", func() {
+				framework.CreatePropagationPolicy(karmadaClient, policyHigherPriority)
+				waitPropagatePolicyReconciled(namespace, policyHigherPriorityName)
```

Comment:
in `Combined Case 3`, the `policyHigherPriority` is expected to be not lazy, so:
```suggestion
                                policyHigherPriority.Spec.ActivationPreference = ""
				framework.CreatePropagationPolicy(karmadaClient, policyHigherPriority)
```

besides, since it is taking effect immediately, we may not need `waitPropagatePolicyReconciled`